src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Types/TypeParameterType.cs

```diff
@@ -1,35 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@dagood - have you ever seen @dotnet-maestro-bot deleting a file before?  I can't tell if I messed this up, or if it did.

Reply:
Not sure what happened here, and it doesn't look like your fault... I don't see anything weird in the build except that somehow it ended up deleting that file: https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=1151441&_a=summary.

https://github.com/dotnet-maestro-bot/corefx/commit/5ef93dda63bed702054e0bba19abd8df940442fd

Reply:
Yea, it's weird, I've never seen anything like that before.  Maybe something to keep our eyes open.

I fixed it up.

Reply:
Something strange is going on in git.  In https://github.com/dotnet/corefx/pull/23278, @JonHanna accidentally managed to check in two files differing only by case.  In https://github.com/dotnet/corefx/pull/25239, I then deleted the erroneous one.  Maybe @dotnet-maestro-bot is somehow interpreting that as needing to delete the other one?  This remaining file with the correct casing definitely shouldn't be deleted.

Reply:
Combining that with the complexity of VSTS trying to minimally clean out the source dir (rather than just removing it entirely), I could imagine that being the cause. Unsure if this might mean other build machines have "poisoned" Git checkouts now that could trigger this issue again in the future.

Reply:
I'm doing a quick repro attempt and I'll file an issue on core-eng if I get one.